daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/value/ValueVersion.scala

```diff
@@ -30,11 +30,15 @@ private[lf] object ValueVersions
   private[value] val minContractIdV1 = ValueVersion("7")
 
   // Older versions are deprecated https://github.com/digital-asset/daml/issues/5220
-  // We force output of recent version, but keep reading older version as long as
-  // Sandbox is alive.
-  val SupportedStableVersions = VersionRange(ValueVersion("6"), ValueVersion("6"))
+  val SupportedStableVersions: VersionRange[ValueVersion] =
+    VersionRange(ValueVersion("6"), ValueVersion("6"))
```

Comment:
By the usage in `object Repl`, one attempting to assemble sensible arguments to  could be easily misled into thinking that versions < 6 are not "supported stable versions", and so remove the "unsupported" versions as mentioned in my overview. Likewise one could make the same assumption from the definition of `EngineConfig.Stable` regarding transaction versions < 10.

It should be clearer that this is not the case. In other words, practically speaking, how deep does one have to dig to access "really supported stable input versions"?